README.md

```diff
@@ -125,6 +125,20 @@ Additional notes:
 - `InstanceName.address` can be used instead of numbers (in the expected, calldata, storage) to insert the contract instance address
 - If the deployer call is not specified for some instance, it will be generated automatically with empty calldata.
 
+
+## Ethereum Solidity semantic tests format
+
+The format of these test files is that of the Ethereum Solidity  [semantic tests](https://github.com/ethereum/solidity/tree/develop/test/libsolidity/semanticTests).
+Test descriptions and expected results are embedded as comments in the test file; lines begin with “`//`” for Solidity files, and “`#`” for Vyper. The beginning of the test description is indicated by a comment line containing “`----`”.
+
+For a new test file to be run by the `era-compiler-tester` integration test framework, its name must be added to the index file for its superdirectory, either `solidity/ethereum/index.yaml` or `vyper/ethereum/index.yaml`, followed by a colon and an `enabled: true` line.  `hash:` and `version:` fields are optional.
```

Comment:
Actually only `modes`, `enabled`, and `hash` fields supposed to be edited manually there.
The index itself with names and hashes must only be updated with `tests-updater`.
I don't know how `tests-updater` will behave if there is a manually added entry.

Reply:
Separated manual addition (vs. `tests-updater`) in 97ca927.